server/repository/assessment/updateCountry.ts

```diff
@@ -14,12 +14,12 @@ export const updateCountry = async (
   return client.one<Country>(
     `
         update ${assessmentCycleName}.country
-        set props = props || $2
+        set props = $2
         where country_iso = $1
         returning *;
 
     `,
-    [countryIso, JSON.stringify(country.props)],
+    [countryIso, country.props],
```

Comment:
no stringify needed?

Reply:
works without

Reply:
regarding this i wrote in dev:

how............... can this be possible?
https://github.com/openforis/fra-platform/pull/1239/files#diff-6e90bef548c0033f2e457073433be4dbfdbcc2c5858af6a0e04ce397a3a850dbR17
[7:02](https://fraplatform.slack.com/archives/C565LS99U/p1649433734517249)
This only works if i dont concat the object ( || $2 )
[7:04](https://fraplatform.slack.com/archives/C565LS99U/p1649433854850279)
If I concat:

    Updating country.props.status from editing to review works
    Updating country.props.desk_study from false to true does not work....?